course/functions.php

```diff
@@ -85,3 +85,25 @@ function course_register_block_patterns_category() {
 
 add_action( 'init', 'course_register_block_patterns_category' );
 
+add_filter(
+	'theme_lesson_templates',
+	function( $page_templates, $theme, $post ) {
```

Comment:
We should try to avoid using inline functions with filters (and actions). It makes it next to impossible to remove them when you need to using `remove_filter`.

Reply:
Thanks @donnapep , I've moved it to a separate function here https://github.com/Automattic/themes/pull/7265/commits/6ad7328bb94b4f3b15b8219dbcf0e4a2de3d13b0